test/run/ch-convert.bats

```diff
@@ -326,49 +326,59 @@ test_from () {
 
 @test 'ch-convert: --no-clobber' {
     # ch-image
-    printf 'FROM alpine:3.9\n' | ch-image build -t tmpimg -f - "$BATS_TMPDIR"
+    printf 'FROM alpine:3.17\n' | ch-image build -t tmpimg -f - "$BATS_TMPDIR"
     run ch-convert --no-clobber -o ch-image "$BATS_TMPDIR" tmpimg
     echo "$output"
     [[ $status -eq 1 ]]
     [[ $output = *"error: exists in ch-image storage, not deleting per --no-clobber: tmpimg" ]]
 
+    # convert ch_timg into ch-image format
+    ch-image delete timg | 'true'
+    if [[ $(file "$ch_timg") = *"symbolic link"* ]]; then
```

Comment:
I'd use `stat(3)` here instead. There are output format options to get just what you want. `file(3)` is too broad.